src/main/java/at/hannibal2/skyhanni/features/garden/farming/FarmingWeightDisplay.kt

```diff
@@ -239,7 +251,7 @@ object FarmingWeightDisplay {
             }
         }
 
-        val totalWeight = (localWeight + weight)
+        val totalWeight = (localWeight + weight - shWeightDiff)
```

Comment:
this formular `localWeight + weight - shWeightDiff` gets used in three spots in the code now. please use a function